### ISSUE TITLE
Allow loot system to change mob loot depending on if the mob is on fire or not.

### ISSUE THREAD

**Issue by ItzASecret** [PluginBugs/Issues-ItemsAdder#2596]
### Terms

- [X] I'm using the very latest version of ItemsAdder and its dependencies.
- [X] I already searched on this [Github page](https://github.com/PluginBugs/Issues-ItemsAdder/issues) to check if the same suggestion was already reported.
- [X] I already searched on the [plugin wiki](https://itemsadder.devs.beer/) to know if the feature is already implemented.
- [X] I already searched on the **Discord** server to know if anyone already has a solution for this or if the feature was already suggested.

### Discord tag (optional)

ItzASecretBoi#1300

### Describe the solution you'd like

Players, (typically speedrunners) will light mobs on fire, and kill them, too automatically cook the food. Well, I realized that doing that with my edible IA items from mobs doesn't really provide the same result.

So basically, allow loot to change if mob dropping it is on fire.

### Is your feature request related to a problem?

Not a big issue, it just takes away from that "vanilla" experience.

### Describe alternatives you've considered

None.

### Additional context

Go LoneDev!

**Comment by LoneDev6** [comment]
Good idea
